libia2/main.c

```diff
@@ -27,6 +27,15 @@ __attribute__((naked)) int __wrap_main(int argc, char **argv) {
       "call __real_main\n"
       // Restore the old stack pointer before returning.
       "mov main_sp(%%rip), %%rsp\n"
+      // Save return value
+      "mov %%rax,%%rbx\n"
```

Comment:
`__wrap_main` needs to preserve %rbx. Can't we use one of 'rdi, rsi, rdx, rcx, r8, r9, r10, r11' instead?